### PR TITLE
don't send trace from FormNotFoundException in edit_form_attr (show message on UI)

### PR DESCRIPTION
**Pull request by lwyszomi**
@czue 
http://manage.dimagi.com/default.asp?92320#1150692
screen shot 2016-05-24 at 11 01 22

### CODE REVIEW

File: corehq/apps/app_manager/views/forms.py [hunk 1]
```diff
@@ -227,103 +227,111 @@ def _edit_form_attr(request, domain, app_id, unique_form_id, attr):
 
     """
 
-    app = get_app(domain, app_id)
-    form = app.get_form(unique_form_id)
```
Comment (czue) [inline corehq/apps/app_manager/views/forms.py]: I'm guessing it's just this line that's failing - could we make the try/catch just around it?

Comment (lwyszomi) [inline corehq/apps/app_manager/views/forms.py]: @czue 👍

File: corehq/apps/app_manager/views/forms.py [hunk 2]
```diff
@@ -227,103 +227,111 @@ def _edit_form_attr(request, domain, app_id, unique_form_id, attr):
...
+
+        handle_media_edits(request, form, should_edit, resp, lang)
+
+        app.save(resp)
+        if ajax:
+            return HttpResponse(json.dumps(resp))
+        else:
+            return back_to_main(request, domain, app_id=app_id, unique_form_id=unique_form_id)
+    except FormNotFoundException as e:
+        messages.error(request, unicode(e))
```
Comment (czue) [inline corehq/apps/app_manager/views/forms.py]: I think we can remove this if it was an ajax request (hopefully the calling code handles errors properly). for a non-ajax request (if that's possible?) can we just say "there was an error saving, please try again"

**Comment by lwyszomi** [comment]
@czue Updated

File: corehq/apps/app_manager/views/forms.py [hunk 3]
```diff
@@ -227,13 +227,20 @@ def _edit_form_attr(request, domain, app_id, unique_form_id, attr):
...
     resp = {}
 
+    app = get_app(domain, app_id)
+    try:
+        form = app.get_form(unique_form_id)
+    except FormNotFoundException as e:
+        if ajax:
+            return HttpResponseBadRequest(unicode(e))
+        else:
+            messages.error(request, "There was an error saving, please try again!")
```
Comment (czue) [inline corehq/apps/app_manager/views/forms.py]: can you add a translation for this?

**Comment by czue** [comment]
one more comment, then :+1: on :white_check_mark:

File: corehq/apps/app_manager/views/forms.py (hunk 3 above)
Comment (czue) [inline corehq/apps/app_manager/views/forms.py]: (translation markup, not translation)

**Comment by lwyszomi** [comment]
@czue Added

**Comment by lwyszomi** [comment]
@czue 
Could you kick off the build?

**Comment by millerdev** [comment]
Not sure if test failure is related to this change.
```
ERROR: test_get_call_center_domains (corehq.apps.callcenter.tests.test_utils.CallCenterDomainTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/callcenter/tests/test_utils.py", line 347, in test_get_call_center_domains
    _create_domain('cc-dom1', True, True, 'flw')
  File "/mnt/corehq/apps/callcenter/tests/test_utils.py", line 376, in _create_domain
    doc=domain.to_json(),
  File "/mnt/corehq/elastic.py", line 54, in send_to_elasticsearch
    doc_exists = doc_exists_in_es(index, doc_id)
  File "/mnt/corehq/elastic.py", line 44, in doc_exists_in_es
    return get_es_new().exists(es_meta.index, es_meta.type, doc_id)
  File "/vendor/lib/python2.7/site-packages/elasticsearch/client/utils.py", line 69, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/vendor/lib/python2.7/site-packages/elasticsearch/client/__init__.py", line 290, in exists
    id), params=params)
  File "/vendor/lib/python2.7/site-packages/elasticsearch/transport.py", line 307, in perform_request
    status, headers, data = connection.perform_request(method, url, params, body, ignore=ignore, timeout=timeout)
  File "/vendor/lib/python2.7/site-packages/elasticsearch/connection/http_urllib3.py", line 93, in perform_request
    self._raise_error(response.status, raw_data)
  File "/vendor/lib/python2.7/site-packages/elasticsearch/connection/base.py", line 105, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
TransportError: TransportError(503, u'')
```

**Comment by czue** [comment]
not related and the build before without the translations succeeded so going to merge.

**Pull request merged**